flexget/components/seen/cli.py

```diff
@@ -55,7 +55,7 @@ def seen_forget(manager: Manager, options):
 
 
 def seen_add(manager: Manager, options):
-    DEFAULT_TASK = 'cli_add'
+    default_task = 'cli_add'
```

Comment:
This feels like a constant.

Reply:
According to PEP 8’s [Constants](https://peps.python.org/pep-0008/#constants), constants are defined at the module level, not within functions. This is a constant defined inside a function. Should I use `noqa`, disable the rule, or keep this constant in lowercase?

Reply:
I've come around, I think this instance is fine in lower case.